src/useScrollToBottom.tsx

```diff
@@ -1,9 +1,12 @@
 import { useEffect } from 'react';
 
-export const useScrollToBottom = (onRendered?: () => void) => {
+export const useScrollToBottom = (
+  onRendered?: () => void,
+  isOperationValidated?: boolean,
+) => {
   useEffect(() => {
     if (onRendered) {
       requestAnimationFrame(onRendered);
     }
-  }, [onRendered]);
+  }, [onRendered, isOperationValidated]);
```

Comment:
When operation becomes validated and the component is rendered, be sure to scroll to the bottom so it stays in view.